scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts

```diff
@@ -1155,19 +1155,38 @@ export class DependencyResolverMain {
   /**
    * Return a list of outdated policy dependencies.
    */
-  getOutdatedPkgsFromPolicies({
+  async getOutdatedPkgsFromPolicies({
     rootDir,
     variantPoliciesByPatterns,
     componentPoliciesById,
+    components,
   }: {
     rootDir: string;
     variantPoliciesByPatterns: Record<string, VariantPolicyConfigObject>;
     componentPoliciesById: Record<string, any>;
+    components: Component[];
   }): Promise<OutdatedPkg[]> {
+    const componentModelVersions: ComponentModelVersion[] = [];
+    await Promise.all(
+      components.map(async (component) => {
+        const depList = await this.getDependencies(component);
```

Comment:
I wonder if we need to filter our extension deps here.
as putting them in the root policy won't change then anyway (and they shouldn't be there)

Reply:
Also wonder what will happen with peer deps

Reply:
Regarding peer dependencies.

I have created a component with a peer dependency:
![image](https://user-images.githubusercontent.com/1927579/185785437-9ab60149-0462-403a-a533-78d785feb8c9.png)

but when I import it, it shows the peer as a runtime dependency:
![image](https://user-images.githubusercontent.com/1927579/185785456-68be8d51-7579-4b7e-85bd-0dce03fe252c.png)

Reply:
extensions and peers in the model are skipped by the update command.